DESCRIPTION

```diff
@@ -1,6 +1,6 @@
 Package: epinowcast
 Title: Flexible Hierarchical Nowcasting
-Version: 0.2.0.17000
+Version: 0.2.0.100000
```

Comment:
seems like the automation tripped up a bit?

Reply:
This is just a placeholder to prevent it auto-releasing (which is what happens with no dev version) and to indicate its later version vs any other dev version. Will drop prior to merging.